test/integration/test_runner.sh

```diff
@@ -27,7 +30,8 @@ execute time go build -o "$SWARM_BINARY" ../..
 # Start the docker engine.
 execute docker --daemon --log-level=panic \
 	--storage-driver="$STORAGE_DRIVER" --exec-driver="$EXEC_DRIVER" &
-DOCKER_PID=$!
+
+trap "pkill -f docker" EXIT
```

Comment:
This `DOCKER_PID` was incorrect I think. It was getting a pid, but it wasn't the pid of the docker daemon. I guess it was the pid of eval ?